include/new_game.h

```diff
@@ -1,19 +1,10 @@
 #ifndef GUARD_NEW_GAME_H
 #define GUARD_NEW_GAME_H
 
-#include "global.h"
```

Comment:
Is there any reason to delete this line?

Reply:
It's in the .c file and it's not present in pokeemerald's new_game.h

Reply:
It's still better to be there as a dependency.